src/apps/concepts/pages/ViewConceptsPage.tsx

```diff
@@ -180,21 +182,19 @@ const ViewConceptsPage: React.FC<Props> = ({
     containerType === SOURCE_CONTAINER
```

Comment:
Hope you verified `SOURCE_VERSION_CONTAINER` checking is not required?

Reply:
tested ui, looks like not required. Still adding it for safety